### ISSUE TITLE
github bridge: validation failed on set-status 

### ISSUE THREAD

**Issue by avsm** [docker/datakit#254]
Possibly some missing escaping? I see this on the bridge in some datakit-ci runs:

```
+32582552513us dkt-github [ERROR] API.set-status {mirage/mirage 8f441498c61188720765a5bf5c7a7516c60a6b2a:ci/datakit/Fedora 24[failure] url=https://ci.ocaml.io:8443/ref/mirage/mir
age/tags/v2.6.1 descr=Command "sh" "-c" "docker build --label com.docker.datakit.digest=154af25d46fefb37f551feab390c02bb --label com.docker.datakit.builton=1473287846.62 -t mirag
eci2:154af25d46fefb37f551feab390c02bb --no-cache --rm --force-rm - < /tmp/tmp-2ca5d9f4/Dockerfile" exited with status 1}: 
Github: GitHub API error: 422 Unprocessable Entity (WebDAV) (RFC 4918) -- Validation Failed
> Resource type: Status
  Field: description
  Code: custom
```

**Comment by samoht** [comment]
Unfortunately the Github API does not describe which constraint the status description should respect, so it's a bit hard to come up with a validation function for this.

**Comment by samoht** [comment]
Fixed by #273
